### PR TITLE
Netlify supports TLS 1.3

### PR DESCRIPTION
**Pull request by hampuskraft**
[Deleted for privacy reasons]

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -584,7 +584,7 @@ <h3 id="cdn-paas"><a href="#cdn-paas">CDN &amp; PaaS performance</a></h3>
             <td class="ok">yes</td>
             <td class="ok">yes</td>
             <td class="ok"><a href="https://www.netlify.com/blog/2015/10/20/netlify-news-no.-6/">yes</a></td>
-            <td class="alert">no</td>
+            <td class="ok">yes</td>
```
Comment (igrigorik) [inline index.html]: Are there are docs, announcements, or other references that we can link to?

Comment (reckr) [reply index.html]: Sorry for the late reply, there is no announcements or docs around this, unfortunately. However, visiting a ***.netlify.com** website and inspecting their certificate in a modern browser will tell you that the connection is encrypted over TLS 1.3. Example: https://i.imgur.com/Se2PUpX.png.

Comment (igrigorik) [reply index.html]: Hmm, that's a bummer. I guess we can land it, but the worry is that it's an experimental and/or selectively enabled feature (which I've seen on other CDNs).

**Comment by igrigorik** [comment]
p.s. thanks for the update!

**Pull request merged**